### DISCUSSION TITLE
The Crew - IPTV

### DISCUSSION THREAD

**Discussion by msmith3640** [community/community#19488] [Projects and Issues]
After the latest crew update.... the IPTV functionality no longer works. If I do a fresh install, I have about 30seconds to get to the IPTV menu until it auto updates and then the TV channels are gone.

**Comment by jameswrightbrown-prog** [comment]
I had a similar issue after an update where IPTV channels suddenly disappeared. It might be worth checking whether the update changed or removed support for the playlist source, since that can break functionality. I've seen users with an [iptv abonnement ](https://www.abonnementiptv-officiel.fr/)run into the same problem after automatic updates.

**Comment by awnioussama48-byte** [comment]
I experienced something similar after a recent update. In many cases, the issue isn't with the channels themselves but with changes to the playlist format or player compatibility. It's also worth checking whether the server URL and login details are still valid after the update.

If anyone needs a step-by-step setup guide for different devices, I found this IPTV Smarters Pro guidevery helpful: https://iptvsmarters-officiel.com/smarters-2026/
